evap/student/templates/student_vote.html

```diff
@@ -27,6 +27,19 @@
             </div>
         </div>
     {% endif %}
+    {% if general_errors_exist and not contributor_errors_exist %}
```

Comment:
The whole outer div is duplicated, the only thing that changed is the inner error message, right? In that case, can you please reduce the scope of the `if` to just the error message (-> deduplicate the outer div, just have the `if` around the `blocktrans` tag)?